app/models/proceeding.rb

```diff
@@ -42,13 +44,37 @@ def domestic_abuse?
     ccms_matter_code == "MINJN"
   end
 
-  # def default_level_of_service_level
-  #   "3"
-  # end
-  #
-  # def default_level_of_service_name
-  #   "Full Representation"
-  # end
+  def substantive_scope_limitations
+    scope_limitations.where(scope_type: :substantive)
+  end
```

Comment:
**just a suggestion:** 

if there is only 1 expected and or order does not matter at this point, then could we do a [find_by](https://api.rubyonrails.org/v7.0.3.1/classes/ActiveRecord/FinderMethods.html#method-i-find_by) here, thereby removing need for `.first` below. should probably remove the `s` too in that case 
```suggestion
  def substantive_scope_limitation
    scope_limitations.find_by(scope_type: :substantive)
  end
```

Reply:
As discussed with Joel, this is being implemented to allow multiples so leaving as-is for now.
Similar to the use of `.first` that is the next thing to be removed when we have the multiple handlers done